<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectoryManager</class>
 <widget class="QWidget" name="DirectoryManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>644</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">DirectoryManager QWidget ,
DirectoryManager QLabel,
DirectoryManager QButton {
    border: none;
}

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="_add_dir_btn">
       <property name="text">
        <string>Ajouter un dossier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_del_dir_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Retirer le dossier</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="_stackpane">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeWidget" name="_dir_tree">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="indentation">
          <number>12</number>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="headerStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="2" column="0">
              <widget class="QProgressBar" name="_progress">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximum">
                <number>0</number>
               </property>
               <property name="value">
                <number>0</number>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="format">
                <string>Initialisation</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Vos images sont en cours d'indexage</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QWidget" name="widget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
           <widget class="QPushButton" name="_add_first_btn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Ajouter un dossier</string>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="_ntm">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string notr="true">Il semble que vous n'avez pas encore ajoutez de dossier !</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>_add_dir_btn</sender>
   <signal>clicked()</signal>
   <receiver>DirectoryManager</receiver>
   <slot>addDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_add_first_btn</sender>
   <signal>clicked()</signal>
   <receiver>DirectoryManager</receiver>
   <slot>addDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_del_dir_btn</sender>
   <signal>clicked()</signal>
   <receiver>DirectoryManager</receiver>
   <slot>removeDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_dir_tree</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>DirectoryManager</receiver>
   <slot>onItemSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_dir_tree</sender>
   <signal>itemExpanded(QTreeWidgetItem*)</signal>
   <receiver>DirectoryManager</receiver>
   <slot>onItemExpanded(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_dir_tree</sender>
   <signal>itemCollapsed(QTreeWidgetItem*)</signal>
   <receiver>DirectoryManager</receiver>
   <slot>onItemCollapsed(QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addDirectory()</slot>
  <slot>removeDirectory()</slot>
  <slot>onItemSelected()</slot>
  <slot>onItemExpanded(QTreeWidgetItem*)</slot>
  <slot>onItemCollapsed(QTreeWidgetItem*)</slot>
 </slots>
</ui>
